recipes/prophet/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "prophet" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e682e8ea6e1ee26a92cf289f207d539f30e44879126c128ff8f4e360eb25a8b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  binary_has_prefix_files:
+    - Lib/site-packages/prophet/stan_model/prophet_model.pkl          # [win]
+    - lib/python*/site-packages/prophet/stan_model/prophet_model.pkl  # [unix]
```

Might be able to remove the selector by using [`SP_DIR`]( https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html?highlight=SP_DIR#environment-variables-set-during-the-build-process )

```suggestion
    - {{ SP_DIR }}/stan_model/prophet_model.pkl
```

---

Worked for Linux and OSX but gave `RuntimeError: build/binary_has_prefix_files paths must use / as the path delimiter on Windows`  in win. I'm guessing that SP_DIR uses "\" in windows. I'm trying using SP_DIR for not win but keeping the explicit path for win.